dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -116,3 +116,19 @@ object reflects the newly requested size in `PersistentVolume.Spec.Capacity`. At
 this point, you can create or re-create a new pod from the PVC to finish the
 file system resizing. Once the pod is running, the newly requested size is
 available and `FileSystemResizePending` condition is removed from the PVC.
+
+[[recover_from_resize_failure]]
+== Recover from failure in Expanding volumes
+
+If for some reason expanding underlying storage fails either on master or on node,
+you may want to manually recover PVC state and cancel resize requests that will be
```

Comment:
Suggest changing this to: "you can manually recover the PVC state and cancel the resize requests that are continuously retired by the controller without administrator intervention."